app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+
```

Comment:
- [ ] Great job defining the association in the comment model! 😊 However, in the context of our Entity-Relationship Diagram (ERD), a comment should not only be associated with a post but also with a user. While you've correctly defined the connection between comments and posts, it appears that the link to users is currently missing.

   - To fully align your implementation with the ERD, please make sure to establish the necessary association between comments and users as well. This will help maintain data integrity and ensure that every comment can be properly attributed to the user who created it. 

Happy coding! 👍🚀